sotodlib/tod_ops/fft_ops.py

```diff
@@ -356,6 +359,18 @@ def calc_wn(aman, pxx=None, freqs=None, low_f=5, high_f=10):
     if pxx is None:
         pxx = aman.Pxx
 
+    # limit upper frequency cutoffs to hwp freq
+    if 'hwp_angle' in aman._fields:
```

Comment:
`'hwp_angle' in aman` should return true if it exists in its fields. So there is no need to specifically check in `_fields`